maestro-android/src/androidTest/java/dev/mobile/maestro/Media.kt

```diff
@@ -0,0 +1,32 @@
+package dev.mobile.maestro
+
+import android.content.ContentValues
+import android.provider.MediaStore
+import androidx.test.platform.app.InstrumentationRegistry
+import java.io.OutputStream
+
+object MediaStorage {
+
+    fun getOutputStream(mediaName: String, mediaExt: String): OutputStream? {
+        val uri = when (mediaExt) {
+            Service.FileType.JPG.ext,
+            Service.FileType.PNG.ext,
+            Service.FileType.GIF.ext,
+            Service.FileType.JPEG.ext -> MediaStore.Images.Media.EXTERNAL_CONTENT_URI
+            Service.FileType.MP4.ext -> MediaStore.Video.Media.EXTERNAL_CONTENT_URI
+            Service.FileType.MP3.ext -> MediaStore.Audio.Media.EXTERNAL_CONTENT_URI
+            else -> throw IllegalStateException("mime type not yet supported")
```

Comment:
nit : could also output `mediaExt` here

`"mime type $mediaExt not yet supported"`

Reply:
Done ✅